### PR TITLE
[86endm2hw] contact page

### PR DESCRIPTION
**Pull request by cl7-dp2**
- contact form create 
- first name and last name 
- with email address

### CODE REVIEW

**Comment by pathusutariya** [comment]
Title is wrong It's not as per we discussed.
Title should be 
**[86endm2hw] Contact Page**

File: contact.html
Comment (pathusutariya) [inline contact.html:1]: Bootstrap file included. but classes are missing from form and everywhere. Please use proper class and styles.

**Review by pathusutariya — CHANGES_REQUESTED** [review]
Please use bootstrap class into Form

File: contact.html
Comment (cl7-dp2) [reply contact.html:1]: Now Changes Done use bootsrap class for form

**Review by cl7-dp2 — COMMENTED** [review]
changes thay gya che

**Review by cl7-dp2 — COMMENTED** [review]
Code Formatted Complete

File: contact.html
Comment (pathusutariya) [inline contact.html:1]: Form element is missing from form.

**Review by pathusutariya — CHANGES_REQUESTED** [review]
`<form>` element is missing from Form.

File: contact.html
Comment (cl7-dp2) [reply contact.html:1]: add form element in contact page

**Review by cl7-dp2 — COMMENTED** [review]
complete changes with add form tag

**Review by pathusutariya — APPROVED** [review]
Good to go

**Pull request merged**